relational_operators/TextScanOperator.cpp

```diff
@@ -18,6 +18,11 @@
  **/
 
 #include "relational_operators/TextScanOperator.hpp"
+#include "relational_operators/RelationalConfig.h"
```

Comment:
Can you sort the above two included files alphabetically? We follow a convention that lists of included files are sorted. If you are dealing with a large list of included files, you may use your IDE's sorting feature, instead of manual effort.